contract/deploy/ensDeployer.ts

```diff
@@ -83,6 +86,124 @@ const f = async function (hre: HardhatRuntimeEnvironment) {
     UniversalResolver: await ensDeployer.universalResolver(),
     Multicall: await Multicall.address
   }) + '\'')
+  // Additional configuration data
```

Comment:
Move this and below to separate files? They seem to be exclusively for testing locally. It probably shouldn't be in deploy folder

Reply:
Done now separated.
This was for local testing, however it would also be good to have some tools for testing dns servers on mainnet.
For that I created [this branch](https://github.com/jw-1ns/ens-deployer/tree/test-mainnet-dns) using a private key (rather than a mnemonic).